src/node_modules.cc

```diff
@@ -100,10 +100,19 @@ const BindingData::PackageConfig* BindingData::GetPackageJSON(
     return &cache_entry->second;
   }
 
+  auto missing_cache_entry = binding_data->missing_package_configs_.find(path.data());
+  if (missing_cache_entry != binding_data->missing_package_configs_.end()) {
```

Comment:
You can just use unordered_set.contains() functionality.

Reply:
Wasn't sure if C++20 was allowed. Done.